wagtail/models/i18n.py

```diff
@@ -174,13 +174,182 @@ def is_active(self) -> bool:
+
+        If a model inherits from
+        :py:class:`DraftStateMixin <wagtail.models.DraftStateMixin>`,
+        then a translation of an instance in the original queryset could exist, but
+        be in a draft state (i.e. ``live=False``). By default, draft translations are
+        not considered when compiling the translated instances and the original instance
+        is used in the localized queryset. To override this behavior and allow draft
+        translations in the localized queryset, pass
+        ``include_draft_translations=True``. Regardless of this option, the resulting
+        localized queryset will be the same length as the original.
+
+        Note: The draft state of an original instance is not considered. I.e., if the
+        original instance is in draft, its translation is also in draft and
+        ``include_draft_translations=False`` is used, then the original instance will
+        be used in the localized queryset (regardless of its draft state). To exclude
+        original draft instances from the localized queryset, filter the original
+        queryset to only include live instances.
+
+        By default, the localized queryset can contain untranslated instances from the
+        original queryset if no translation for the instance is available. If this is
+        not desired, pass ``include_only_translations=True`` to only include instances
+        that are translated into the active locale. In case not all instances are
+        translated and ``include_draft_translations=True`` is passed, the resulting
+        queryset will be shorter than the original.
+
+        Note: If localization is disabled via the ``WAGTAIL_I18N_ENABLED`` setting, this
+        method returns the original queryset unchanged.
+
+        Note: Because this method uses a complex query to retrieve the items in the
+        localized queryset, effects of methods like ``annotate``, ``select_related`` and
+        ``prefetch_related`` are not retained. If you want to use them on the resulting
+        queryset, you should apply them after ``.localized()``. However, you can use
+        `annotate` on the original queryset and use its value for filtering. Just be
+        aware that the annotate column will not be available on the returned queryset.
+        """
+        # Skip localization if i18n is not enabled. This behavior is consistent with
+        # the behavior of the `localized` property on `TranslatableMixin`.
+        if not getattr(settings, "WAGTAIL_I18N_ENABLED", False):
+            return self
+
+        # Get all instances that are available in the active locale. We can find these
+        # by getting all model instances that have a translation key from the original
+        # queryset and that are available in the active locale.
+        active_locale = Locale.get_active()
+        original_translation_keys = self.values_list("translation_key", flat=True)
+        translated_instances = self.model.objects.filter(
```

Comment:
Sorry to drop in unannounced, but shouldn't references to the model manager be made to the default manager? I.e. to `_default_manager` and not to `objects`, in case they are not the same:

```
translated_instances = self.model._default_manager.filter(
```

Reply:
Good shout. I followed what I have seen other places, but I am happy to give this a go.